<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RubricItemGradeWidget</class>
 <widget class="QWidget" name="RubricItemGradeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="rubricItemMax">
   <property name="geometry">
    <rect>
     <x>438</x>
     <y>4</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="rubricItemGrade">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>14</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="rubricItemDescription">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>4</y>
     <width>351</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>427</x>
     <y>4</y>
     <width>16</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
